llap-common/src/gen/protobuf/gen-java/org/apache/hadoop/hive/llap/daemon/rpc/LlapDaemonProtocolProtos.java

```diff
@@ -1,6 +1,7 @@
 // Generated by the protocol buffer compiler.  DO NOT EDIT!
 // source: LlapDaemonProtocol.proto
 
+// Protobuf Java Version: 3.25.5
```

Comment:
Is this a manual change or an auto-generated?
If manual, why is it needed?

Reply:
This is auto-generated code. There is a `protobuf` profile that generates it.